docs/walkthrough_transcribe.md

```diff
@@ -2,22 +2,90 @@
+
+![image: Correction of the transcription.](img/transcribe/correction.gif "Correction of the transcription")
+
+!!! Note
+    To follow the changes made on the transcription line, you can click on it and select "+Toggle history" in which you can see the changes made.
+
+![image: History of the transcription.](img/transcribe/toggle_history.gif "History of the transcription")
 
 ## Virtual Keyboard
 
-## Compare transcriptions
+!!! warning
+    The section needs to be completed.
+
+A virtual keyboard is available in the "Text" pane in the "Edit" tab. To activate it, click on the icon "Toggle Virtual Keyboard for this document", then click on one of the line.  
+The virtual keyboard lets you predefine a series of keyboard shortcuts and virtual buttons to facilitate the writing of frequent special caracters. The "Manage Keyboard" button allows the management of this setting, and it is also possible to import a new configuration via a JSON file or a link to a JSON file.
+
+## Versions of transcription
+
+### Manage transcription
+A document can be associated simultanously to several versions of transcription. Each version has its own name. There is always a default one named "manual". A new version is created as soon as a text recognition model is applied to the document or a transcription is imported. The version is then named after the applied model, the name given to the import or the import name by default, "Zip Import".  
+To display the list of versions of transcription available, click on the drop-down menu in the top-right corner and choose the version you want to see.
+
+![image: Versions of transcription.](img/transcribe/transcription_version.gif "Versions of transcription")
+
+To delete a transcription version, click on the "Transcription management" button (blue button with the gear) in the top-right corner. The "Delete" button will delete the chosen version irrevocably for all the images of the document.
+
+![image: Deletion of transcription.](img/transcribe/delete_version.gif "Deletion of transcription")
+
+!!! Warning
+    Even when a text recognition model is applied to only part of the images in a document, the entirety of the document-parts will be associated to this new version of the transcription. This means that, even if the display is at document-part level, the creation or **deletion** of a transcription version will impact all the images of the document.
+
+### Compare transcriptions
+It is possible to compare several versions of a transcription. To do so, click on the "Transcription management" button (blue button with the gear) in the top-right corner. Choose the versions you want to compare by ticking the checkboxes under "Compare", then close the pop-up box, by clicking on the cross or simply by clicking somewhere on the page. Then, select any line in the "Transcription" pane and click on "+Toggle transcription comparison" to see the difference.  
```

Comment:
As discussed earlier, I don't know if you should mention the aspect of the button ("blue button with the gear") since it might change. 

You are missing one point: you need to define the base version used for the comparison, which is basically the transcription version displayed in the window. I don't exactly know how to describe that. If you want to switch to another base version, you just need to change which transcription version is displayed on screen before opening the transcription pop-up.